coredns/manifest.json

```diff
@@ -66,10 +66,11 @@
     },
     "dashboards": {
       "CoreDNS": "assets/dashboards/coredns.json"
```

Comment:
missing a comma here, it should look like this:
`"CoreDNS": "assets/dashboards/coredns.json",`
 it makes the json file invalid.